components/qmp6988/qmp6988.h

```diff
@@ -0,0 +1,197 @@
+
+#ifndef __QMP6988_H__
+#define __QMP6988_H__
+
+#include <stdbool.h>
+#include <i2cdev.h>
+#include <esp_err.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define QMP6988_U16_t uint16_t
+#define QMP6988_S16_t int16_t
+#define QMP6988_U32_t uint32_t
+#define QMP6988_S32_t int32_t
+#define QMP6988_U64_t uint64_t
+#define QMP6988_S64_t int64_t
+
+#define QMP6988_I2C_ADDR_GND 0x70
+#define QMP6988_I2C_ADDR_VDD 0x56
+
+#define QMP6988_CHIP_ID 0x5C
+
+#define QMP6988_CHIP_ID_REG         0xD1
+#define QMP6988_RESET_REG           0xE0 /* Device reset register */
+#define QMP6988_DEVICE_STAT_REG     0xF3 /* Device state register */
+#define QMP6988_CTRLMEAS_REG        0xF4 /* Measurement Condition Control Register */
+#define QMP6988_PRESSURE_MSB_REG    0xF7 /* Pressure MSB Register */
+#define QMP6988_TEMPERATURE_MSB_REG 0xFA /* Temperature MSB Reg */
+
+#define SUBTRACTOR 8388608
```

Comment:
Please remove these macros from the header. Here they simply pollute the global namespace.
The same goes for defining registers.